subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -360,29 +362,43 @@ uint8_t ll_cig_parameters_commit(uint8_t cig_id, uint16_t *handles)
 			 * directions
 			 */
 			if (tx) {
+				uint32_t iso_interval_adjust_us;
 				uint8_t max_pdu;
 				uint8_t bn;
 
 				bn = cis->lll.tx.bn;
 				max_pdu = cis->lll.tx.max_pdu;
-				set_bn_max_pdu(cis->framed, iso_interval_us,
-					       cig->c_sdu_interval,
-					       cis->c_max_sdu, &bn, &max_pdu);
+				iso_interval_adjust_us =
+					set_bn_max_pdu(cis->framed, iso_interval_us,
+						       iso_interval_cig_us, cig->c_sdu_interval,
+						       cis->c_max_sdu, &bn, &max_pdu);
+				if (iso_interval_adjust_us != iso_interval_us) {
+					iso_interval_us = iso_interval_adjust_us;
+
+					goto ll_cig_parameters_commit_retry;
+				}
 				cis->lll.tx.bn = bn;
 				cis->lll.tx.max_pdu = max_pdu;
 			} else {
 				cis->lll.tx.bn = 0U;
 			}
 
 			if (rx) {
+				uint32_t iso_interval_adjust_us;
```

Comment:
Just a thank you for postfixing the unit to the variables :) Makes it a lot easier to follow IMO

Reply:
Yeah, the spec doesn't make it easy either. Lots of switching between us, ms and 1.25ms units :D